Dockerfile.dist

```diff
@@ -142,6 +142,10 @@ COPY --from=download /download/cosign /usr/local/bin/cosign
 # your dist directory, not just the freshly built binaries.)
 COPY --from=build /build/dist/ /usr/local/bin/
 
+# Gzip them because that's what the cli downloader image expects, see
+# https://github.com/securesign/sigstore-ocp/blob/main/images/Dockerfile-clientserver
+RUN gzip /usr/local/bin/ec_*
```

Comment:
Wait! Isn't this going to **replace** the EC binaries with a gzipped version? That is not going to work when someone uses the ec-cli image in a Tekton Task, for example.

Reply:
In line 146 we copy the one binary that matches the os/arch of the container we're building. So we should be fine.

Reply:
I mean line 150, see below.

Reply:
That one won't get gzipped.

Reply:
Aha! Thanks